Publisher/nl/productupdate20230511.md

```diff
@@ -0,0 +1,35 @@
+# Productupdate - Opvolgactie verbeteringen, uitschrijfbevestiging en verzendmomenten uitsluiten met RRule
+
+## Opvolgactie op basis van veldwijziging
+Het is nu mogelijk om in het _bestemming checken_ blok te kijken of een veld is aangepast. Hierbij kan je aangeven dat de opvolgactie uitgevoerd moet worden als het veld naar een specifieke waarde is aangepast of bij iedere wijziging waarbij de waarde niet gespecificeerd is. 
+
+Na het toevoegen van het blok, kies je bij _vergelijking_ voor _is bijgewerkt_ voor iedere wijziging of voor _is bijgewerkt naar_ voor een specifieke wijziging. Bij deze laatste optie krijg je een invoerveld om de waarde te specificeren.
```

Comment:
Nadat je het blok hebt toegevoegd, kun je bij _vergelijking_ kiezen voor de optie _is bijgewerkt_ als je wilt dat de opvolgactie wordt uitgevoerd bij elke wijziging in het veld. Als je wilt dat de opvolgactie alleen wordt uitgevoerd wanneer het veld is bijgewerkt naar een specifieke waarde, kies dan de optie 'is bijgewerkt naar' en voer de waarde in het invoerveld in.